src/information-panel/information-panel.scss

```diff
@@ -17,7 +17,6 @@
   box-sizing: border-box;
   max-width: 100%;
   max-height: 100%;
-  z-index: 2; // Needs to be higher than table's column resizers.
```

Comment:
Good to remove as long as we [isolate within the header](https://github.com/iTwin/iTwinUI/pull/632/files#r869425199).